WordPress/WordPressTest/WPFeedbackGeneratorTests.swift

```diff
@@ -0,0 +1,121 @@
+import XCTest
+@testable import WordPress
+
+// MARK: - Enum mapping tests
+
+@available(iOS 10, *)
+class WPFeedbackGeneratorTests: XCTestCase {
+    func testNotificationFeedbackGeneratorMapping() {
+        let successType = WPNotificationFeedbackType.Success
+        XCTAssertEqual(successType.rawValue, successType.systemFeedbackType.rawValue)
```

Comment:
If these asserted against `UINotificationFeedbackType` directly their intention would be explicit.